project/Build.scala

```diff
@@ -6,13 +6,13 @@ object Build extends Build {
   val org = "com.sksamuel.elastic4s"
   val appVersion = "1.7.5"
 
-  val ScalaVersion = "2.11.7"
-  val ScalatestVersion = "2.2.5"
+  val ScalaVersion = "2.12.6"
+  val ScalatestVersion = "3.0.5"
   val MockitoVersion = "1.9.5"
-  val JacksonVersion = "2.6.1"
-  val Slf4jVersion = "1.7.12"
-  val ScalaLoggingVersion = "2.1.2"
-  val ElasticsearchVersion = "1.7.2"
+  val JacksonVersion = "2.9.6"
+  val Slf4jVersion = "1.8.0-beta2"
```

Comment:
@FlorinAcsinte, can you elaborate on why you updated to the beta version of slf4j? We would like to use the 1.7.x branch with scala 2.12, but this specific dependency upgrade blocks us.